packages/solidjs/src/errorboundary.ts

```diff
@@ -0,0 +1,31 @@
+import { captureException } from '@sentry/browser';
+import type { Component, JSX } from 'solid-js';
+import { mergeProps, splitProps, untrack } from 'solid-js';
+import { createComponent } from 'solid-js/web';
```

Comment:
Is `solid-js/web` always available to us? I don't see it being listed as peerDependency or dependency in the SDK's package.json

Reply:
I believe it's available through `solid-js`. At least I didn't have any issues developing locally. The solidjs templates (these are used to start projects ala `create-react-app`) also don't specifically list it as dep, e.g. [here](https://github.com/solidjs/templates/blob/main/ts-vitest/package.json).

Reply:
Ahh I see, thx for explaining! Let's leave it as is and if we get error reports (I have a feeling pnpm could complain 👀 ) we can add it later.